### ISSUE TITLE
Consistent Synthetic Sequences Unlock Structural Diversity in Fully Atomistic De Novo Protein Design

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#637]
```bibtex
@misc{reidenbach2025,
  title = {Consistent Synthetic Sequences Unlock Structural Diversity in Fully Atomistic De Novo Protein Design},
  author = {Reidenbach, Danny and Cao, Zhonglin and Zhang, Zuobai and Didi, Kieran and Geffner, Tomas and Zhou, Guoqing and Tang, Jian and Dallago, Christian and Vahdat, Arash and Kucukbenli, Emine and Kreis, Karsten},
  year = {2025},
  eprint = {2512.01976},
  archivePrefix = {arXiv},
  primaryClass = {q-bio.BM},
  url = {https://arxiv.org/abs/2512.01976},
  note = {Version 3}
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **Low self-consistency under a second structure predictor does not establish that AFDB sequence–structure pairs are physically inconsistent** [@reidenbach2025]. Figure 2 reports 26.6% backbone and 19.1% all-atom recovery with ESMFold, whereas Section 3 reports approximately 65% on a subset evaluated with ColabFold. This is strong evidence of predictor-dependent disagreement but insufficient evidence of experimental misfolding, contradicting the paper’s stronger “sequence–structure misalignment” interpretation. Related: [[Nearly half of CATH domains do not pass designability filters used to evaluate protein backbone design performance]]; [[Structure prediction neural networks trained on the same structural dataset are likely to predict the same conformation]]; [[Protein folding neural networks do not learn the physics of protein folding for most proteins]]. Placement: append to [[Nearly half of CATH domains do not pass designability filters used to evaluate protein backbone design performance]].

- **Aligning synthetic sequences with synthetic structures improves in-silico atomistic co-design metrics more than filtering structures for designability alone** [@reidenbach2025]. Table 2 shows that the aligned dataset improves co-designability and structural diversity in two model families, whereas training on refolded structures or a designability-filtered subset does not improve—and sometimes worsens—the same metrics. This supports the benefit of synthetic training data but contradicts the simpler hypothesis that selecting individually designable structures is sufficient. Evidence is moderately strong within these benchmarks, although dataset size, sequence composition, and selection procedure differ between comparisons. Related: [[Training backbone diffusion models on synthetic data improves designability]]; [[Protein models designed using inverse folding can be used to supplement training DBs for PLMs and structure prediction models]]; [[Models trained on synthetic data tend to converge to mean]]. Placement: new atomic note, `Aligning synthetic sequences and structures improves atomistic protein co-design metrics.md`.

- **Consistency-distilled training data can optimize agreement with the model ecosystem used to evaluate it without establishing physical designability** [@reidenbach2025]. Section 3 constructs the dataset using inverse-folding and structure-prediction models, while Appendix G.1 evaluates designability using the same model families. The methodological overlap is explicit, but its effect size is not isolated; no experimental folding or functional validation is reported, and Section H concedes that the method provides no guarantee of function or binding. The missed insight is that Tables 1–2 demonstrate improved evaluator consistency, not biological success. Related: [[Sequence recovery in inverse folding models is not correlated with self-consistency of generated designs]]; [[Protein structure prediction and design metrics don't correlate with expression probability]]; [[Protein structure prediction and design confidence metrics do not correlate with binding affinity]]. Placement: new atomic note, `Consistency distillation can improve evaluator agreement without establishing physical designability.md`.

- **Protein-backbone designability scores depend substantially on how many inverse-folding candidates are evaluated** [@reidenbach2025]. Table 5 and Appendix Section F.1 show that repeated inverse-folding raises the reported fraction of designable backbones by 10–28% relative to a single candidate across the evaluated generators. This is strong computational evidence that designability is partly an evaluation-pipeline property, contradicting interpretations that attribute the score entirely to the backbone generator. Related: [[Increasing diffusion samples is sufficient to yield correctly predicted antibody-antigen complexes]]; [[Sequence recovery in inverse folding models is not correlated with self-consistency of generated designs]]; [[Nearly half of CATH domains do not pass designability filters used to evaluate protein backbone design performance]]. Placement: new atomic note, `Protein backbone designability depends on inverse-folding sample count.md`.

- **Strong backbone-generation performance does not reliably transfer to joint sequence–structure generation** [@reidenbach2025]. Table 6 and Appendix Section F.2.3 show that a leading backbone architecture loses substantial diversity when extended to co-design, while another architecture performs much better on the same dataset. This is strong evidence within the compared architectures that data alignment alone does not determine performance; architecture remains a major factor. Related: [[Joint sequence-structure diffusion or flow matching models show superior performance when designing structure at the beginning and sequence near the end]]; [[Coupling sidechain and main chain prediction or design does not always lead to improvements]]; [[Multimodal sequence-structure guidance outperforms unimodal guidance in cases where both need to be designed]]. Placement: append to [[Coupling sidechain and main chain prediction or design does not always lead to improvements]].

- **Synthetic inverse-folding-derived training sequences have a shifted amino-acid composition relative to natural sequences** [@reidenbach2025]. Figure 6 and Appendix Section B.3 show enrichment of charged residues in the synthetic dataset, which the authors acknowledge may matter downstream. The compositional difference is directly measured, but its contribution to benchmark gains and its experimental consequences are not tested; a missed possibility is that some apparent benefit or loss of transfer arises from this distribution shift rather than sequence–structure alignment itself. Related: [[Glutamate- and lysine-rich designs can fail due to expression failures resulting from adenosine-rich sequences]]; [[ProteinMPNN-derived inverse folding methods underdesign aromatic residues]]; [[Models trained on synthetic data tend to converge to mean]]. Placement: append to [[Glutamate- and lysine-rich designs can fail due to expression failures resulting from adenosine-rich sequences]].

#### Metadata check

The full arXiv v3 PDF, including its appendices, was accessible and reviewed. The issue title, author list, year, arXiv identifier `2512.01976`, and `reidenbach2025` citekey match the source; v3 was last revised December 11, 2025. The citekey and paper are absent from `bibliography.bib`. No journal DOI is reported, but the canonical arXiv DOI is `10.48550/arXiv.2512.01976`; it should be added to the eventual bibliography entry.
